Classes/MessageBus.swift

```diff
@@ -55,7 +55,7 @@ private struct Observation {
     
     @objc public func post(_ event: PKEvent) {
         DispatchQueue.global().async { [weak self] in
-            PKLog.info("Post event: \(event)")
+            PKLog.info("Post event: \(String(describing: type(of: event)))")
```

Comment:
there is no originator here, the originator is the class the post came from and we don't hold reference to it here. we already print in trace on the player engine when events are fired. do you think of any specific scenario which this will help?